### DISCUSSION TITLE
Comparison to Firefox's internal Cookie Banner Blocker

### DISCUSSION THREAD

**Discussion by schewara** [OhMyGuus/I-Still-Dont-Care-About-Cookies#11741] [General]
As Firefox rolled out an internal [Cookie Banner Blocker](https://support.mozilla.org/en-US/kb/cookie-banner-reduction) feature, I was wondering how this project overlaps?

By also leveraging https://github.com/mozilla/cookie-banner-rules-list, maybe some duplicate efforts in adding new sites could be eliminated as well if feasible.

**Comment by aiolos01** [comment]
The new version (123) doesn't have it. They're only testing it in Germany so who knows? Seems similar but it's all about how many sites it supports.
